README.md

```diff
@@ -395,6 +396,22 @@ client.ref_data_isin(['US0378331005', 'US5949181045'], mapped: true) # {'US03783
 
 See [#ISIN Mapping](https://iexcloud.io/docs/api/#isin-mapping) for detailed documentation or [isin_mapping.rb](lib/iex/resources/isin_mapping.rb) for returned fields.
 
+### Get Symbols
+
+Returns an array of symbols
+
+```ruby
+symbols = client.ref_data_symbols()
+
+symbols.first.exchange # NAS
```

Comment:
I know we have this elsewhere, but let's improve it here and there while we're here:

```ruby
symbols = ...

symbol = symbols.first
symbol.iex_id # => 'IEX_46574843354B2D52'
symbol.region 
symbol.symbol
```

you get the idea ...